appveyor.yml

```diff
@@ -20,6 +20,11 @@ notifications:
     on_build_status_changed: false
 
 install:
+# If there's a newer build queued for the same PR, cancel this one
```

Comment:
Good to know. Since this is in the AppVeyor YAML I'll turn off rolling builds in the AppVeyor settings. (Btw if you ever need to make changes to AppVeyor settings for anything, I made you a collaborator on my AppVeyor account a long time ago, so feel free to change things as you see fit.)